# Reason: some newer versions of makeinfo errors out on @sp4
# Source: oe-lite
# Upstream: fixed in newer versions of nano

diff --git a/doc/texinfo/nano.texi b/doc/texinfo/nano.texi
index fe12428..6eb95d5 100644
--- a/doc/texinfo/nano.texi
+++ b/doc/texinfo/nano.texi
@@ -32,7 +32,7 @@ This manual documents GNU @code{nano}, a small and friendly text
 editor.
 
 This manual is part of the GNU @code{nano} distribution.@*
-@sp4
+@sp 4
 Copyright (C) 1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007,
 2008, 2009 Free Software Foundation, Inc.
 
